edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/IMetricsGauge.cs

```diff
@@ -3,6 +3,6 @@ namespace Microsoft.Azure.Devices.Edge.Util.Metrics
 {
     public interface IMetricsGauge
     {
-        void Set(long value, string[] labelValues);
+        void Set(double value, string[] labelValues);
```

Comment:
I initially was using a gauge and exposing the availability ratio (between 0 and 100). I don't need this change anymore, but it makes sense to leave it since the underlying library only takes a double, so it will always be converted anyway.